app/src/main/res/values/strings.xml

```diff
@@ -8,9 +8,9 @@
 
     <!-- Favorites -->
     <string name="button_favorites">Favorites</string>
-    <string name="button_select_favorites">Select favorites</string>
+    <string name="button_select_favorites">Add more</string>

```

Comment:
I think "select" (or a synonym) would be more accurate because the dialog opened by this button allows to both add and remove favorites.

Reply:
```suggestion
    <string name="button_select_favorites">Select…</string>

```